src/app/modules/rup/components/elementos/adjuntarDocumento.html

```diff
@@ -86,4 +86,12 @@
             </div>
         </div>
     </div>
-</form>
\ No newline at end of file
+</form>
+
+<ng-container>
```

Comment:
el <ng-container> sin condicional o for esta de más, hay que sacarlo. 
También fiajte que en Boostrap si tenes una `class="col"`, antes deberías tener una `class="row"`.

https://v4-alpha.getbootstrap.com/layout/grid/

Ver si es necesario usar encolumnado

Reply:
@liquid36  resuelto lo del encolumnado , no comprendo lo del sin condicional o for a que hace referencia .

Reply:
`<ng-container>` por lo general se usa así:
```
<ng-container *ngIf="hola">
    ....
</ng-container>
```

Tenerlo así solo no cumple ninguna función.